docs/metadata/Git/git-troubleshooting.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Git Troubleshooting
+id: git-troubleshooting
+description: How to troubleshoot permissions and network issues with Git
+tags: []
+---
+
+Sometimes there is an error when Prophecy tries to reach Git. This could be caused by your user's [authorization](#debugging-git-credentials), a [repository setting](#debugging-repository-setttings), or a [network](#debugging-network-issues) issue. Fear not! This guide outlines what to do if you see error messages such as:  
```

Comment:
We should use second person, so "This could be cause by your [authorization]..."